### ISSUE TITLE
[bitnami/harbor] imageChartStorage.caBundleSecretName

### ISSUE THREAD

**Issue by obeyler** [bitnami/charts#1654]
**Which chart**:
bitnami/harbor 
**Description**
bitnami/harbor is based on goHarbor helm chart
int his chart they allow to specify the CA for external S3
https://github.com/goharbor/harbor-helm/blob/master/values.yaml#L170
Could you provide the same way to be able to use https with selflcertificate on minio ?

**Comment by javsalgar** [comment]
Hi,

Let me see if I understood it correctly. You want that ChartMuseum can accept a custom CA, right? I think we can improve the configurability of ChartMuseum by allowing custom configuration files or setting extra environment variables. We will take a look and update the ticket when we add these improvements. Thanks for reporting.

**Comment by obeyler** [comment]
Yes you have well undertood my need. Our Minio is exposed on HTTPS (signed certificate by our own CA). Chartmuseum need to accept this CA. I see that on latest go Harbor they add this behavior.

**Comment by MaiReo** [comment]
Access UI /harbor/projects/1/helm-charts GET internal server error
![image](https://user-images.githubusercontent.com/11777709/85870530-b76ae180-b7ff-11ea-8231-b4b141c09f33.png)

Should Core need to accept custom CA, too?

2020/06/26 14:38:27.909 [D] [middleware.go:52]  |  192.168.3.100| 500 | 370.772714ms|   match| GET      /api/chartrepo/maireo/charts   r:/api/chartrepo/:repo/charts
2020-06-26T14:38:27Z [ERROR] [/server/error/error.go:53]: {"errors":[{"code":"INTERNAL_SERVER_ERROR","message":"fail to list charts: RequestError: send request failed\ncaused by: Get \"https://192.168.3.253:9000/harbor?prefix=maireo\": x509: certificate signed by unknown authority"}]}
/bitnami/blacksmith-sandox/harbor-core-2.0.0/src/github.com/goharbor/harbor/src/server/error/error.go:81, github.com/goharbor/harbor/src/server/error.apiError

**Comment by javsalgar** [comment]
Hi,

Could you provide the values you used for deploying the chart? Which version did you use?

**Comment by carrodher** [comment]
Unfortunately, this issue was created a long time ago and although there is an internal task to fix it, it was not prioritized as something to address in the short/mid term. It's not a technical reason but something related to the capacity since we're a small team.

Being said that, contributions via PRs are more than welcome in both repositories (containers and charts). Just in case you would like to contribute.

During this time, there are several releases of this asset and it's possible the issue has gone as part of other changes. If that's not the case and you are still experiencing this issue, please feel free to reopen it and we will re-evaluate it.
